sdk/schemaregistry/azure-schemaregistry-avroserializer/CHANGELOG.md

```diff
@@ -0,0 +1,5 @@
+# Release History
+
+## 1.0.0b1 (Unreleased)
+
+Version 1.0.0b1 is the first preview of our efforts to create a user-friendly and Pythonic client library for Azure Schema Registry Avro Serializer. 
```

Comment:
Might be worth leaving one or two bulletpoints on what the core functionalities are (perhaps under the **Features** subheader) to give someone guidance if they land on this changelog.

Reply:
added some snippets to show what we provide